pass-model/src/main/java/org/dataconservancy/pass/model/Publisher.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2018 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataconservancy.pass.model;
+
+import java.net.URI;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Describes a Publisher and its related Journals, also the path of it's participation in PubMedCentral
+ * @author Karen Hanson
+ * @version $Id$
+ */
+
+public class Publisher extends PassEntity {
+
+    @JsonProperty("@type")
+    protected String type = PassEntityType.PUBLISHER.getName();
+    
+    /** Name of publisher */
+    public String name;
+    
+    /** URIs of Journals associated with publisher */
+    public List<URI> journals;

```

Comment:
Do these fields need to be public?  Why isn't `getJournals()` sufficient

Reply:
oops!